pysr/sr.py

```diff
@@ -1989,6 +1989,72 @@ def get_hof(self):
             return ret_outputs
         return ret_outputs[0]
 
+    def plot_eqn_histogram(self, ax=None, threshold=0.05, fontsize=10, **plt_kwargs):
+        """Create a plot of the best equations.
+
+        Parameters
+         ----------
+        ax : Axes, default=None
+             The axes to draw to
+        threshold : float, default=0.05
+             The plotting threshold. Plots only equations that with a score
+             higher than threshold*max_score
+        fontsize: float, default=10
+             Font size for plotting
+        plt_kwargs : dict, default=None
+            Dictionary of extra arguments to pass for plotting
+        """
+
+        assert threshold > 0, "Invalid threshold"
+        assert threshold < 1, "Invalid threshold"
+
+        if ax is None:
+            import matplotlib.pyplot as plt
+            ax = plt.gca()
```

Comment:
I think `plt.gca()` just gets the current/last axis, rather than creating a new one. But I think we would want this to generate a new figure/axis.